db/migrate/20230822190037_create_comments.rb

```diff
@@ -0,0 +1,12 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.datetime :CreatedAt
+      t.datetime :UpdatedAt
+      t.references :user, null: false, foreign_key: true
+      t.references :post, null: false, foreign_key: true
+
+      t.timestamps
```

Comment:
- [ ] [Optional] [Highly recommended] Please remove columns `:CreatedAt` and `:UpdatedAt` because `t.timestamps` at the end will create it for you automatically.